csharp/downgrades/193c35b0e3776f8ee9a49e984e393974b4c670f6/old.dbscheme

```diff
@@ -245,7 +245,7 @@ tokens(
               | @destructor | @callable_accessor | @value_or_ref_type | @declaration_with_accessors
               | @local_function;
 
-/** LOCATIONS, ASEMMBLIES, MODULES, FILES and FOLDERS **/
+/** LOCATIONS, ASSEMBLIES, MODULES, FILES and FOLDERS **/
```

Comment:
Please revert all changes to files in `csharp/downgrades` and `csharp/upgrades`.